src/test/java/org/springframework/samples/petclinic/repository/jpa/TestJpaOwnerRepository.java

```diff
@@ -1,86 +1,83 @@
 package org.springframework.samples.petclinic.repository.jpa;
 
-
-import static org.mockito.Mockito.*;
-import static org.mockito.BDDMockito.given;
-
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
-
-import javax.persistence.EntityManager;
-import javax.persistence.PersistenceContext;
-import javax.persistence.Query;
 import javax.transaction.Transactional;
 
 import org.junit.jupiter.api.Test;
-import org.mockito.Mock;
-import org.mockito.Mockito;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.samples.petclinic.model.Owner;
 import org.springframework.samples.petclinic.repository.OwnerRepository;
-import org.springframework.stereotype.Repository;
 import org.springframework.test.annotation.Rollback;
 import org.springframework.test.context.ActiveProfiles;
-import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit.jupiter.SpringJUnitConfig;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
-import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.mockito.Mockito.*;
+
+import java.util.ArrayList;
+
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
-//@RunWith(SpringJUnit4ClassRunner.class)
 @SpringJUnitConfig(locations= {"classpath:spring/business-config.xml","classpath:spring/mvc-test-config.xml"})
 @ActiveProfiles({"jpa","HSQLDB"})
 public class TestJpaOwnerRepository {
 
-	
-	
 	@Autowired
 	OwnerRepository ownerRepository;
-	
-	
+
+	private Owner getOwnerObject() {
+		Owner owner=new Owner();
+		owner.setLastName("David");
+		owner.setFirstName("Test");
+		owner.setAddress("test");
+		owner.setCity("test");
+		owner.setId(1);
+		owner.setTelephone("1234567899");
+		
+		return owner;
+	}
+
 	@Test
 	@Transactional
 	@Rollback(true)
-	public void testFindById1() {
-		Owner owner = ownerRepository.findById(1);

```

Comment:
Use the owner object saved previously ( usually in a common place setUp ) or save owner here and use its id here.